src/main/java/mate/academy/service/AuthenticationService.java

```diff
@@ -10,7 +10,12 @@ public class AuthenticationService {
      * @return true if user by email exists and passed password is equal to user's password.
      * Return false in any other cases.
      */
+    private static final UserService userService = new UserService();
+
     public boolean login(String email, String password) {
+        if (userService.findByEmail(email) != null) {
+            return userService.findByEmail(email).getPassword().equals(password);
+        }
         return false;
```

Comment:
It is still can be simplified)
You can reuse the result of findByEmail method, so there is no need to call it twice

Reply:
Sorry for the clarification.
I need to create a variable that finds and returns a user. And further, make manipulation with this variable. For example: 
```
User user =  userService.findByEmail(email);
        if (user != null) {
            return user.getPassword().equals(password);
        }
        return false;
```